Index: paraview-3.2.2/Applications/Client/CMakeLists.txt
===================================================================
--- paraview-3.2.2.orig/Applications/Client/CMakeLists.txt	2008-06-04 23:40:01.000000000 +0200
+++ paraview-3.2.2/Applications/Client/CMakeLists.txt	2008-06-15 12:30:23.718122813 +0200
@@ -458,53 +458,6 @@
     COMPONENT Runtime)
 ENDIF(NOT Q_WS_MAC)
 
-# Install system libraries on Linux (will not work on any other Unix
-# yet)
-FOREACH(fmpglib avcodec avformat avutil)
-  IF(FFMPEG_${fmpglib}_LIBRARY)
-    GET_FILENAME_COMPONENT(FFMPEG_LIB_DIR_tmp ${FFMPEG_${fmpglib}_LIBRARY} PATH)
-    GET_FILENAME_COMPONENT(FFMPEG_LIB_NAME_tmp ${FFMPEG_${fmpglib}_LIBRARY} NAME)
-    FILE(GLOB FFMPEG_LIB_LIST RELATIVE ${FFMPEG_LIB_DIR_tmp} "${FFMPEG_${fmpglib}_LIBRARY}*")
-    INSTALL(CODE "
-MESSAGE(STATUS \"Installing \${CMAKE_INSTALL_PREFIX}/${PV_INSTALL_LIB_DIR_CM24}/${FFMPEG_LIB_NAME_tmp}\")
-EXECUTE_PROCESS (WORKING_DIRECTORY ${FFMPEG_LIB_DIR_tmp}
-                 COMMAND tar c ${FFMPEG_LIB_LIST}
-                 COMMAND tar -xC \${CMAKE_INSTALL_PREFIX}/${PV_INSTALL_LIB_DIR_CM24})
-        " COMPONENT Runtime)
-    #INSTALL(FILES ${FFMPEG_${fmpglib}_LIBRARY} DESTINATION ${PV_INSTALL_LIB_DIR_CM24})
-    
-  ENDIF(FFMPEG_${fmpglib}_LIBRARY)
-ENDFOREACH(fmpglib)
-
-SET (QTLIBLIST QTCORE QTGUI QTNETWORK QTXML QTTEST)
-# the variable and library names are not the same
-IF (WIN32)
-  SET (QTLIBLIST ${QTLIBLIST} QTASSISTANTCLIENT)
-ELSE (WIN32)
-  SET (QTLIBLIST ${QTLIBLIST} QTASSISTANT)
-ENDIF (WIN32)
-
-IF(NOT Q_WS_MAC)
-  FOREACH(qtlib ${QTLIBLIST})
-    IF (NOT WIN32)
-      #INSTALL(FILES ${QT_${qtlib}_LIBRARY_RELEASE} DESTINATION ${PV_INSTALL_LIB_DIR_CM24})
-      GET_FILENAME_COMPONENT(QT_LIB_DIR_tmp ${QT_${qtlib}_LIBRARY_RELEASE} PATH)
-      GET_FILENAME_COMPONENT(QT_LIB_NAME_tmp ${QT_${qtlib}_LIBRARY_RELEASE} NAME)
-      FILE(GLOB QT_LIB_LIST RELATIVE ${QT_LIB_DIR_tmp} "${QT_${qtlib}_LIBRARY_RELEASE}*")
-      INSTALL(CODE "
-MESSAGE(STATUS \"Installing \${CMAKE_INSTALL_PREFIX}/${PV_INSTALL_LIB_DIR_CM24}/${QT_LIB_NAME_tmp}\")
-EXECUTE_PROCESS (WORKING_DIRECTORY ${QT_LIB_DIR_tmp}
-                 COMMAND tar c ${QT_LIB_LIST}
-                 COMMAND tar -xC \${CMAKE_INSTALL_PREFIX}/${PV_INSTALL_LIB_DIR_CM24})
-        " COMPONENT Runtime)
-    ELSE (NOT WIN32)
-      GET_FILENAME_COMPONENT(QT_DLL_PATH_tmp ${QT_QMAKE_EXECUTABLE} PATH)
-      INSTALL(FILES ${QT_DLL_PATH_tmp}/${qtlib}4.dll DESTINATION ${PV_INSTALL_BIN_DIR_CM24} COMPONENT Runtime)
-    ENDIF (NOT WIN32)
-    
-  ENDFOREACH(qtlib)
-ENDIF(NOT Q_WS_MAC)
-
 # Hard-coded install rules for anything left to install. These should
 # probably go to a local cmake file. Change to match the build system's
 # libraries
@@ -517,21 +470,6 @@
     DESTINATION ${PV_EXE_INSTALL_CM24} 
     PERMISSIONS OWNER_WRITE OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ 
     COMPONENT Runtime)
-ELSE(WIN32)
-  IF (NOT Q_WS_MAC)
-    IF(PV_NEED_SHARED_FORWARD)
-      INSTALL(FILES ${QT_BINARY_DIR}/assistant 
-        DESTINATION ${PV_EXE_INSTALL_CM24} 
-        PERMISSIONS OWNER_WRITE OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ 
-        COMPONENT Runtime
-        RENAME assistant-real)
-    ELSE(PV_NEED_SHARED_FORWARD)
-      INSTALL(FILES ${QT_BINARY_DIR}/assistant 
-        DESTINATION ${PV_EXE_INSTALL_CM24} 
-        PERMISSIONS OWNER_WRITE OWNER_READ OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ 
-        COMPONENT Runtime)
-    ENDIF(PV_NEED_SHARED_FORWARD)
-  ENDIF (NOT Q_WS_MAC)
 ENDIF(WIN32)
 
 # Call this one last so that it happens after the above targets are all installed:
